frontend/src/components/admin-page/components/button/actions-button.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import styles from './styles.module.scss';
+import { IUser } from 'common/interfaces';
+import Button from '../../../common/button/button';
+import { ButtonColor, ButtonIcon, ButtonStyleType } from 'common/enums';
+
+interface IProps {
+  edit: (user?: IUser) => void;
+  deleteUser: (id: string) => void;
```

Comment:
```suggestion
  onUserEdit: (user: IUser) => void;
  onDeleteUser: (id: string) => void;
```
You repeat this type aa lot. It would be better to create a type for it